api/ai.py

```diff
@@ -1,9 +1,14 @@
+import os
 import time
 from datetime import datetime, timezone
 from invoke_types import InvocationRequest, Actor, LLMMessage
-from settings import MODEL, MODEL_KEY, MAX_TOKENS
+from settings import MODEL, MODEL_KEY, MAX_TOKENS, INFERENCE_SERVICE, API_KEY, OLLAMA_URL, GROQ_API_BASE, OPENROUTER_API_BASE
 import json
 import anthropic
+import openai
+from dotenv import load_dotenv
```

Comment:
nit: we should already have .env loaded bc we import `settings` which calls `load_dotenv`. doesn't really matter tho

Reply:
good point!